products/jenkins.md

```diff
@@ -32,7 +32,14 @@ releases:
 -   releaseCycle: "2"
     outOfOrder: true # to keep it at the top
     releaseLabel: "Regular"
-    releaseDate: 2016-04-20
+    releaseDate: 2024-11-12
+    eol: false
+    latest: "2.485"
+    latestReleaseDate: 2024-11-12
+
+-   releaseCycle: "2.484"
+    releaseLabel: "Regular"
+    releaseDate: 2024-11-05
     eol: false
     latest: "2.484"
     latestReleaseDate: 2024-11-05
```

Comment:
why do we need separate releaseCycle for 2.484?

Reply:
I don't think we need that, that was a miskate, fixed!